README.md

```diff
@@ -262,6 +262,21 @@ console.log("categoriesResult: ", categoriesResult);
 
 **NOTE: In the next major version release, path parameters will be single encoded by default**
 
+## Documentation
```

Comment:
Can we add these details in the main doc -index.html at the top level? That way we can see the instructions as soon as we open the doc